### ISSUE TITLE
Versioncode of 2.0.3?

### ISSUE THREAD

**Issue by ghost** [Berlin-Vegan/berlin-vegan-guide#68]
Tag 2.0.3 is referring to version 2.0.3 with versioncode 24, but one commit after that, d9e6a1a2f90f42ee63ea905fb2703662ff8c44e6 bumps the VC. Maybe this was an erroneous tag? If so remove 2.0.3 and repush the tag for 2.0.3 set to the right commit.

**Comment by smeir** [comment]
During our release, I tag and merge the "development2 branch with "master", set a tag on master and then increase version in development branch, not to forget it. Maybe there are better and easier solutions. I don't know. :-(

**Comment by jmastr** [comment]
This is correct behavior. Check out our README.md:

...
- check is everything is fine and especially check the version string under 'About'
- if everything is fine, push to upstream:

```
git push origin master --tags
```
- switch to development branch and increase versionCode in "app/build.gradle"

We developed VC 24, we released it, and now we develop VC 25 til next release.

One can argue if this makes sense or not but it is our convention.

Closing the issue.
